app/assets/javascripts/discourse/app/instance-initializers/register-service-worker.js

```diff
@@ -0,0 +1,8 @@
+import { registerServiceWorker } from "discourse/lib/register-service-worker";
+
+export default {
+  initialize(owner) {
+    let { serviceWorkerURL } = owner.lookup("service:session");
+    registerServiceWorker(serviceWorkerURL);
```

Comment:
This file is moved. The function used to take the `owner` as the first argument, but it's unused so I removed it.